test/utility/misc.c

```diff
@@ -139,6 +139,17 @@ testutil_cleanup(TEST_OPTS *opts)
 }
 
 /*
+ * testutil_check_long --
+ *	Check if WT_DISABLE_LONG_TESTS is set and exit if so
+ */
+void
+testutil_check_long(void)
+{
+	if (getenv("WT_DISABLE_LONG_TESTS") != NULL)
```

Comment:
Is getenv going to work on non-posix platforms? Do any of our tests work on non-posix platforms? I'd be inclined to use ``__wt_getenv`` instead.